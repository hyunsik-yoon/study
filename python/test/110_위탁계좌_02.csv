거래일자,거래번호,거래구분,적요명,종목코드,종목명,수량,단가,거래금액,수수료,거래세,소득세,농특/주민세,세금합계,정산금액,미수변제/연체,기타대여/연체,유가잔고,예수금금잔,대용잔고,약정금액,대출(상환)금액,대출이자,대체금융회사,대체계좌번호,계좌명,받는분표시내용,보내는분표시내용,과표
2023-07-01,2,매수-코스닥,매수,A005290,동진쎄미켐,145,"28,550","4,139,750","20,580",0,0,0,0,"4,160,330",0,0,145,"8,566",0,"4,139,750",0,0,,,,,,
2023-07-13,1,예탁금이용료,예탁금이용료(정기),,,0,0,10,0,0,0,0,0,10,0,0,0,"8,576",0,0,0,0,,,,,,(KRW)10
2023-07-20,2,매수-장내,매수,A285130,SK케미칼,25,"161,500","4,037,500","20,070",0,0,0,0,"4,057,570",0,0,25,"81,413",0,"4,037,500",0,0,,,,,,
2023-07-20,1,매도-코스닥,매도,A005290,동진쎄미켐,145,"28,700","4,161,500","20,690","10,403",0,0,"10,403","4,130,407",0,0,0,"4,138,983",0,"4,161,500",0,0,,,,,,
2023-07-27,2,매수-코스닥,매수,A059090,미코,457,"13,250","6,055,250","30,110",0,0,0,0,"6,085,360",0,0,457,848,0,"6,055,250",0,0,,,,,,
2023-07-27,1,매도-장내,매도,A285130,SK케미칼,25,"242,000","6,050,000","30,080","6,050",0,"9,075","15,125","6,004,795",0,0,0,"6,086,208",0,"6,050,000",0,0,,,,,,
2023-08-07,2,매수-장내,매수,A102460,이연제약,13,"21,100","274,300","1,360",0,0,0,0,"275,660",0,0,13,"7,549,530",0,"274,300",0,0,,,,,,
2023-08-07,1,매도-코스닥,매도,A059090,미코,457,"17,250","7,883,250","39,200","19,708",0,0,"19,708","7,824,342",0,0,0,"7,825,190",0,"7,883,250",0,0,,,,,,
2023-08-24,1,매도-장내,매도,A102460,이연제약,13,"17,650","229,450","1,140",229,0,344,573,"227,737",0,0,0,"227,917",0,"229,450",0,0,,,,,,
0000-01-01,,,,,이 테스트는 종목별 매수 매도가 한쌍씩 존재하는데 뒤섞여져 있는 테스트,,,,,,,,,,,,,,,,,,,,,,,
