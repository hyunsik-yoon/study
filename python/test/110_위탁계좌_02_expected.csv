매수거래일자,매도거래일자,종목명,거래수량,총수수료,총세금합계,매수단가,매도단가,매수거래금액,매도거래금액,순수익
2023-07-01,2023-07-20,동진쎄미켐,145,41270,10403,28550,28700,4139750,4161500,-29923
2023-07-20,2023-07-27,SK케미칼,25,50150,15125,161500,242000,4037500,6050000,1947225
2023-07-27,2023-08-07,미코,457,69310,19708,13250,17250,6055250,7883250,1738982
2023-08-07,2023-08-24,이연제약,13,2500,573,21100,17650,274300,229450,-47923

